arlo-client/src/components/AuditForms/CalculateRiskMeasurement.tsx

```diff
@@ -292,6 +296,17 @@ const CalculateRiskMeasurement: React.FC<IProps> = ({
     }
   }
 
+  const [ballots, setBallots] = useState<IBallot[]>([])
+  useEffect(() => {
```

Comment:
is this nesting of an async function inside the `useEffect` callback necessary? What's the purpose?

Reply:
@benadida It's a bit of a gotcha with `useEffect` actually. If you make the callback in the `useEffect` async itself then it automatically returns a `Promise<null>`. [But `useEffect` uses the return value of its callback function as a cleanup function](https://reactjs.org/docs/hooks-effect.html#effects-with-cleanup). So you can't return something else from it without complaints. Hence having an async IIFE inside of it instead. 🙂